src/puppetlabs/puppetdb/http/fact_names.clj

```diff
@@ -3,25 +3,28 @@
             [puppetlabs.puppetdb.query.paging :as paging]
             [puppetlabs.puppetdb.jdbc :refer [with-transacted-connection]]
             [net.cgrand.moustache :refer [app]]
-            [puppetlabs.puppetdb.middleware :refer [verify-accepts-json validate-query-params wrap-with-paging-options validate-no-query-params]]
+            [puppetlabs.puppetdb.middleware
+             :refer [verify-accepts-json validate-query-params wrap-with-paging-options validate-no-query-params]]
```

Comment:
this is getting long -- maybe worth using an alias at this point instead of referring the functions themselves